// serverless-esbuild/dist/tests/pack.test.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const pack_1 = require("../pack");
describe('filterFilesForZipPackage', () => {
    it('should filter out files for another zip package', () => {
        expect((0, pack_1.filterFilesForZipPackage)({
            files: [
                {
                    localPath: '__only_service-otherFnName/bin/imagemagick/include/ImageMagick/magick/method-attribute.h',
                    rootPath: '/home/capaj/repos/google/search/.esbuild/.build/__only_service-otherFnName/bin/imagemagick/include/ImageMagick/magick/method-attribute.h',
                },
                {
                    localPath: '__only_service-fnName/bin/imagemagick/include/ImageMagick/magick/method-attribute.h',
                    rootPath: '/home/capaj/repos/google/search/.esbuild/.build/__only_service-fnName/bin/imagemagick/include/ImageMagick/magick/method-attribute.h',
                },
            ],
            depWhiteList: [],
            fnNameWithoutStage: 'service-fnName',
            isGoogleProvider: false,
            hasExternals: false,
            includedFiles: [],
            excludedFiles: [],
        })).toMatchInlineSnapshot(`
      Array [
        Object {
          "localPath": "__only_service-fnName/bin/imagemagick/include/ImageMagick/magick/method-attribute.h",
          "rootPath": "/home/capaj/repos/google/search/.esbuild/.build/__only_service-fnName/bin/imagemagick/include/ImageMagick/magick/method-attribute.h",
        },
      ]
    `);
    });
});
